salt/salt.spec

```diff
@@ -682,6 +685,12 @@ Requires:       python-base
 %endif
 %endif
 
+# In case of SLE15SP7+ no more python3-salt anymore
+%if 0%{?sle_version} >= 150700
+Obsoletes:      python3-salt < %{version}-%{release}
+Provides:       python3-salt = %{version}-%{release}
```

Comment:
Apparently, autobuild team doesn't like to use "Obsoletes/Provides" combo in this situation. They suggest to use "weakremover" or "Conflicts".

Reply:
It seems we cannot sort this out with the suggested "Conflicts" nor "weakremover"

I've just updated this PR, using different obsoletes approach, actually in line to what we did in the past when moving away from "python2-salt".

This should be in sync to latest submission: https://build.suse.de/request/show/367026